packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.types.ts

```diff
@@ -46,6 +46,34 @@ export interface IScrollablePaneProps extends React.HTMLAttributes<HTMLElement |
   initialScrollPosition?: number;
 
   scrollbarVisibility?: ScrollbarVisibility;
+
+  stickyAboveContainerBehavior?: IStickyContainerBehavior;
+  stickyBelowContainerBehavior?: IStickyContainerBehavior;
+}
+
+export interface IStickyContainerBehavior {
+  /**
+   * If true, it replicates actual element instead of keeping placeholder
+   * Best Practices: Perf reasons
+   */
+  notUsePlaceHolder: boolean;
+  /**
+   * If true, arranges stickies based on 'order' prop
+   */
+  arrangeStickiesBasedOnOrder: boolean;
+
+  /**
+   * If it is provided, stickies' stickyPosition must not be 'Both'
+   */
+  containerBehavior: StickyContainerBehaviorType;
+}
+export enum StickyContainerBehaviorType {
```

Comment:
>StickyContainerBehaviorType [](start = 12, length = 27)

Shouldn't this be on individual Stickies?

Reply:
stickies which have same sticky position must 've same behavior - 1. sticky on scroll, 2. sticky always, 3. default, i.e., the prop is at sticky container level

Reply:
Worth throwing an Error if the above is not true in debug builds to help inform devs?

Reply:
ok